### DISCUSSION TITLE
有流言称，部分浏览器会审查此项目的一些链接

### DISCUSSION THREAD

**Discussion by gledos** [WeNeedHome/SummaryOfLoanSuspension#460] [General]
![photo_2022-07-15_12-04-39](https://user-images.githubusercontent.com/23720620/179148169-0eef9e59-bdf2-4380-9933-4f4444ac3726.jpg)

![photo_2022-07-15_12-04-43](https://user-images.githubusercontent.com/23720620/179148166-8331a831-2f44-4ec4-9b5a-8bcd9674dc4e.jpg)

尚不明确真实性。

**Comment by yalo-ddaa** [comment]
这不是流言，国产app一直都这样，上次的996.icu也是被屏蔽了。甚至Firefox也提示“该网页被举报为诈骗网站”。

**Reply by gledos** [reply]
嗯，主要是我没有这种浏览器来测试，这个消息又没有附上具体被审查的 URL，所以我就先用流言一词来描述这个审查了。

**Reply by xiaorong61** [reply]
火狐不完全算国外的浏览器，因为它有国内分公司的。

**Reply by gledos** [reply]
> 火狐不完全算国外的浏览器，因为它有国内分公司的。

是啊，而且火狐国际版发现大陆 IP 后，会限制安装一些去广告的插件……

**Reply by Edan233** [reply]
Safari呢

**Comment by KisaraKira** [comment]
是真实的，国产软件基本上都带这种“反诈”功能的，一旦用户访问这类违禁网站或URL将直接拦截并上报，背后原因令人暖心。

**Reply by lzq2357** [reply]
全球变暖的一大原因就是暖心的事情太多了

**Reply by hktk1024** [reply]
> 全球变暖的一大原因就是暖心的事情太多了

笑“不活”了

**Comment by JACWATA** [comment]
一切影响和谐的，都不准发

**Comment by NultiluK** [comment]
这时候得抱紧chrome等美帝浏览器的大腿了

**Comment by zhw173** [comment]
前两年一直用夸克，大概一年前发现部分收藏的网站打不开并提示那个信息，用其他浏览器就能正常打开

**Comment by C-Joey** [comment]
用chrome呗

**Comment by lionwill** [comment]
夸克和微信打不开了，chrome可以

**Comment by chenjiasu** [comment]
360浏览器不行，进不来页面，
